scripts/globals/mobskills/vulcan_shot.lua

```diff
@@ -19,7 +19,7 @@ end
 
 function onMobWeaponSkill(target, mob, skill)
     local dmgmod = 8
```

Comment:
dmgmod is a multiplier. So this script is multiplying mobs weapon wdmg by 4 in the line you changed and total damage by 8 after that when dmgmod gets used in the global. Changing either one will influence the dmg totals greatly. Both are kinda high but I've no idea how the move is calculated to arrive at its damage numbers on retail.